geohash_hilbert/_utils.py

```diff
@@ -31,27 +31,28 @@ def neighbours(code, bits_per_char=6):
     """Get the neighbouring geohashes for `code`.
 
     Look for the north, north-east, east, south-east, south, south-west, west,
-    north-west neighbours. If you are at the edge of the grid (lng \in (-180, 180),
-    lat \in (-90, 90)), then it wraps around the glob and gets the corresponding neighbor.
+    north-west neighbours. If you are at the east/west edge of the grid
+    (lng \in (-180, 180)), then it wraps around the globe and gets the corresponding
```

Comment:
```suggestion
    (lng ∈ (-180, 180)), then it wraps around the globe and gets the corresponding
```

The `\i` error is new since the last flake8 run on this file. Maybe we use the correct unicode character here.